azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/compute/options/AzureTemplateOptions.java

```diff
@@ -33,6 +36,7 @@
    private String blob;
    private AvailabilitySet availabilitySet;
    private String availabilitySetName;
+   protected List<Map<String, String>> dataDisks = ImmutableList.of();
```

Comment:
Better create an object to carry the information of the data disks? A `Map` looks quite inconvenient and too error-prone, and also harder to document and use by end users.